pandas/_libs/tslibs/offsets.pyx

```diff
@@ -587,15 +583,17 @@ def shift_quarters(int64_t[:] dtindex, int quarters,
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def shift_months(int64_t[:] dtindex, int months, object day=None):
+def shift_months(int64_t[:] dtindex, int months, object day):
     """
     Given an int64-based datetime index, shift all elements
     specified number of months using DateOffset semantics
 
-    day: {None, 'start', 'end'}
+    day: {None, 'start', 'end', 'business_start', 'business_end'}
```

Comment:
remove the None as a valid argument or is it? also need to fix the code

Reply:
None is valid.